settings.gradle

```diff
@@ -1,4 +1,4 @@
 include ':opensrp-chw'
 
-/*include ':opensrp-chw-core'
-project(":opensrp-chw-core").projectDir = new File("../opensrp-client-chw-core/opensrp-chw-core")*/
+include ':opensrp-chw-core'
```

Comment:
Should this also have been commented out?

Reply:
Yes, it should 👍🏾  . 
I'll push a commit to comment this out